### ISSUE TITLE
Quintic functions are misnamed

### ISSUE THREAD

**Issue by aferriss** [glslify/glsl-easings#6]
Hi there, thanks for providing these functions, such a great resource!

All the quintic functions are misspelled as `qintic` without the u. The filenames are correct however, it's just the functions names that are defined with the typo.

**Comment by bmschmidt** [comment]
This is fixed in https://github.com/glslify/glsl-easings/pull/8.

For glslify, Is it possible/desirable to export the same function under two different names for case current code is using the 'qintic' spelling?
Are there non-glslify cases where the mis-spelling should need to be maintained?

**Comment by rreusser** [comment]
@bmschmidt Yeah, I think that's best. Maybe just move the implementation to the properly named one and then just re-export it in the incorrectly named file? (so that the implementations don't accidentally diverge in case someone maintains it and misses the bugfix)

**Comment by rreusser** [comment]
Also, thanks for working on this!! 👏 🙏  I hang around and try to facilitate maintenance from time to time… I think I have permission to actually deploy these fixes and/or add you as someone who can. Don't hesitate to @-ping me if something needs to happen.
